tasks/agent.py

```diff
@@ -6,5 +6,5 @@ def version(ctx):
     with open("release.json", "r") as release_json_stream:
         version = json.load(release_json_stream)
     print(version['agent_version'])
-    return version['agent_version']
+    return "ofek/rust-old-glibc"
```

Comment:
I should have modified the `release.json` file 🤦 , I won't modify this since the CI is already on the way 

We will revert this right after the merge and https://github.com/DataDog/datadog-agent/pull/18303 is merged